tutorminio/patches/openedx-common-settings

```diff
@@ -2,6 +2,7 @@ DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
 VIDEO_IMAGE_SETTINGS["STORAGE_KWARGS"]["location"] = VIDEO_IMAGE_SETTINGS["STORAGE_KWARGS"]["location"].lstrip("/")
 VIDEO_TRANSCRIPTS_SETTINGS["STORAGE_KWARGS"]["location"] = VIDEO_TRANSCRIPTS_SETTINGS["STORAGE_KWARGS"]["location"].lstrip("/")
 GRADES_DOWNLOAD["STORAGE_KWARGS"]["location"] = GRADES_DOWNLOAD["STORAGE_KWARGS"]["location"].lstrip("/")
+GRADES_DOWNLOAD["STORAGE_KWARGS"]["bucket"] = "{{ MINIO_GRADE_BUCKET }}"
```

Comment:
Can you please explain what will be the impact of this change? Am I right to understand that files stored in the grades download bucket are generated when the instructor generates the course grades? If yes, then you're right that this bucket should be private by default.

Reply:
Hello @regisb, yes, the files stored there are these generated by instructutor like Grades Download, it can have sensitive data